mithril.js

```diff
@@ -218,9 +218,34 @@
 		return list
 	}
 
+	var applyIEPerf = (
+			typeof document !== "undefined" &&
+			typeof document.documentMode === "number"
+		) ||
+		(
+			typeof navigator !== "undefined" &&
+			typeof navigator.userAgent === "string" &&
+			/\bEdge\/\d/.test(navigator.userAgent)
+		)
+
 	function insertNode(parentElement, node, index) {
-		parentElement.insertBefore(node,
-			parentElement.childNodes[index] || null)
+		if (applyIEPerf) {
+			if (!parentElement._children) {
+				parentElement._children = []
```

Comment:
imho, if the goal is to change the order of appends, it may make more sense to change it directly in lines 883-914 than to hack the internals of insertNode